spec/serializers/resource_serializer_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require Rails.root.join('app', 'serializers', 'resource_serializer.rb')
+
+describe ResourceSerializer do
+  it 'returns latest published translation for each language' do
+    resource = Resource.find(2)
+    serializer = ResourceSerializer.new(resource)
+    serialization = ActiveModelSerializers::Adapter.create(serializer)
+
+    latest_translations = serialization.as_json[:data][:relationships][:'latest-translations'][:data]
```

Comment:
I have found it helpful to write my specs in this format:
```
it 'is well written' do
  # the lines needed to prepare for the test
  setup

  # as few of lines as possible to actually run the test
  result = do_something

  # all of the expectations
  expect(result).to be_fancy
  expect(1).to be (0)
end
```
Yummmm... spec sandwich 🍔